### ISSUE TITLE
How to use IOU loss like in FCOS

### ISSUE THREAD

**Issue by adizhol** [facebookresearch/maskrcnn-benchmark#1076]
## ❓ Questions and Help
Hi,

Can I simply send box_regression and regression_targets to the iou loss function...
[loss.py]
`            box_loss = iou_loss_fn(
                box_regression[sampled_pos_inds_subset[:, None], map_inds],
                regression_targets[sampled_pos_inds_subset],
            )
`

The iou_loss function:
[https://github.com/tianzhi0549/FCOS/blob/master/fcos_core/layers/iou_loss.py
](url)

Thanks
